storage/container_storage_interface/persistent-storage-csi-ibm-cloud.adoc

```diff
@@ -0,0 +1,23 @@
+[id="persistent-storage-csi-ibm-cloud"]
+= IBM Cloud CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ibm-cloud
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for IBM Cloud Storage.
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to IBM Cloud storage assets with this feature is enabled, {product-title} installs the IBM Cloud CSI Driver Operator and the IBM Cloud CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
```

Comment:
This sentence is a bit complex. I can't think of a great way to break it up, but mentioning it in case you have ideas. Not critical to merging though :+1: